src/discussions/posts/post-editor/messages.js

```diff
@@ -5,10 +5,14 @@ const messages = defineMessages({
     id: 'discussions.post.editor.type',
     defaultMessage: 'Post type',
   },
-  heading: {
-    id: 'discussions.post.editor.heading',
+  addPostHeading: {
+    id: 'discussions.post.editor.add-post-heading',
     defaultMessage: 'Add a post',
   },
+  editPostHeading: {
+    id: 'discussions.post.editor.edit-post-heading',
+    defaultMessage: 'Edit post',
+  },
   typeDescription: {
```

Comment:
I think for consistency change the key ids to camelCase everywhere like followed in rest of file 
`discussions.post.editor.add-post-heading ---> discussions.post.editor.addPostHeading'`